src/navsections/KustomizationSectionBlueprint/KustomizationSectionBlueprint.ts

```diff
@@ -51,29 +58,28 @@ const KustomizationSectionBlueprint: SectionBlueprint<K8sResource, Kustomization
       }
       return scope.isFolderLoading;
     },
-    isVisible: scope => {
-      return !scope.isInClusterMode;
-    },
     isInitialized: (_, rawItems) => {
       return rawItems.length > 0;
     },
+    isEmpty: (scope, rawItems) => {
+      return scope.isFolderOpen && rawItems.length === 0;
+    },
+    shouldBeVisibleBeforeInitialized: true,
+  },
+  customization: {
+    emptyDisplay: {
+      component: KustomizationSectionEmptyDisplay,
+    },
+    beforeInitializationText: 'Get started by browsing a folder in the File Explorer.',
```

Comment:
this needs to show something else if the current project has no Kustomizations?

Reply:
Yes, it does, the text for when there are no kustomizations in the folder is in the KustomizationSectionEmptyDisplay component

In a future PR when I start adding tests, I will refactor the customization interface to either allow a string or a component on some of it's options. For now the "emptyDisplay" needs a component